store/src/sql/entities/event.rs

```diff
@@ -48,22 +49,39 @@ pub struct EventInsertable {
     cid: Cid,
     /// Whether the event is deliverable i.e. it's prev has been delivered and the chain is continuous to an init event
     deliverable: bool,
-    /// The blocks of the event
-    // could use a map but there aren't that many blocks per event (right?)
-    blocks: Vec<EventBlockRaw>,
+    /// The parsed structure containing the actual Event data.
+    event: unvalidated::Event<Ipld>,
 }
 
 impl EventInsertable {
     /// EventInsertable constructor
-    pub fn new(order_key: EventId, blocks: Vec<EventBlockRaw>, deliverable: bool) -> Self {
-        let cid = order_key.cid().unwrap();
+    pub fn new(
+        order_key: EventId,
+        event_cid: Cid,
```

Comment:
It'd be nice to delete the CID from this struct and just get it from event in the future.

Reply:
yeah, I was lamenting the fact that there's no easy way to get the event CID from the Event struct itself while writing this.